license-scanner/types.ts

```diff
@@ -114,7 +116,7 @@ export class DB {
 }
 
 export type RustCrateScannerOutput = {
-  license: string | null | undefined;
+  license: string | null | undefined | { workspace: true };
```

Comment:
Feels like a realy weird type, and I can't find how it's used.

Reply:
Okay, so now we return `Inheritable<>` from the scanner, which is `{workspace: true}`, but do we somehow use it? 
If we wanted just to avoid crash, why not return null there?  
If we wanted to resolve inherited properties, why didn't we do it in rust scanner, why expose it to ts part?

Reply:
Great points.

I didn't want to resolve inherited properties, I created [an issue](https://github.com/paritytech/license-scanner/issues/56) and didn't want to spend time on it, because this functionality is not being used for years.

But now that I think about it again, it should be rather simply to resolve it now.
I'll take another look at it. If not, I'll return null as you suggest.

Reply:
@mutantcornholio I redid it, now I just return null instead of those inheritable types without using them.

Added a new test target - a crate with inherited properties.